web-admin/tests/bookmarks.spec.ts

```diff
@@ -0,0 +1,261 @@
+import { expect, type Page } from "@playwright/test";
+import { assertUrlParams } from "@rilldata/web-common/tests/utils/assertUrlParams";
+import { test } from "./setup/base";
+import { interactWithTimeRangeMenu } from "@rilldata/web-common/tests/utils/exploreInteractions";
+
+test.describe("Bookmarks", () => {
+  // TODO: use a separate explore to isolate bookmarks
+  test("Create and verify home bookmark", async ({ page }) => {
+    await page.goto("/e2e/openrtb/explore/auction_explore");
+
+    // Select "Last 14 days" as time range
+    await interactWithTimeRangeMenu(page, async () => {
```

Comment:
I see you're just re-using an existing abstraction from `web-local`, but food for thought – Playwright seems to advocate for consolidating these sorts of abstractions into a [Page Object Model](https://playwright.dev/docs/pom).

Reply:
Good point. This what we had started with, I remember some devs had objections to this and we moved to a functional model. Lets do a greater refactor and move such methods in a separate PR.